src/components/DocumentationTopic.vue

```diff
@@ -767,6 +767,14 @@ export default {
 }
 
 /deep/ {
+  h1 {
+    @include font-styles(headline-reduced);
+  }
+
+  h2 {
```

Comment:
the `PrimaryContent.vue` already has the same declaration for an H2 style, should we maybe remove it or?

Reply:
Sure. I'm honestly not sure which place is better at the moment, but it would be helpful to have all of the h1-h6 styles together like this, so I'll probably just remove that one.

Reply:
7ea417ae6fb23da26ecebfdabfc4f54ddb6f40c5